modules/codemod/lib/v8/deprecateDrawer.ts

```diff
@@ -0,0 +1,119 @@
+import {API, FileInfo, Identifier, ImportDeclaration, Options} from 'jscodeshift';
+
+const mainPackage = '@workday/canvas-kit-labs-react';
+const drawerPackage = '@workday/canvas-kit-labs-react/drawer';
+const drawerImportNames = [
+  'Drawer',
+  'DrawerProps',
+  'DrawerHeaderProps',
+  'DrawerHeader',
+  'DrawerDirection',
+];
+
+export default function transformer(file: FileInfo, api: API, options: Options) {
+  const j = api.jscodeshift;
+
+  const root = j(file.source);
+  let hasDrawerImports = false;
+
+  // This toggles the failsafe that prevents us from accidentally transforming something unintentionally.
+  root.find(j.ImportDeclaration, (nodePath: ImportDeclaration) => {
+    const value = nodePath.source.value;
+    // If there's an import from the drawer package, set the import boolean check to true
+    if (value === drawerPackage) {
+      hasDrawerImports = true;
+      return false;
+    }
+
+    // If there's an import from the main package, check to see if Drawer or DrawerProps are among the named imports
+    // e.g. import {Drawer} from '@workday/canvas-kit-react';
+    if (value === mainPackage) {
```

Comment:
Not sure if this is possible since drawer is in `labs` do we still need this check?